game-core/src/main/java/games/strategy/triplea/attachments/RulesAttachment.java

```diff
@@ -198,8 +198,8 @@ private void setBattle(final String value) throws GameParseException {
     }
     if (!s[3].equalsIgnoreCase("currentRound")) {
       try {
-        getInt(s[3].split("-")[0]);
-        getInt(s[3].split("-")[1]);
+        getInt(splitOnHyphen(s[3])[0]);
+        getInt(splitOnHyphen(s[3])[1]);
```

Comment:
I was tempted to refactor this to avoid the control flow by exception.  However, there is [zero code coverage](https://codecov.io/gh/triplea-game/triplea/src/master/game-core/src/main/java/games/strategy/triplea/attachments/RulesAttachment.java#L199...206) here.  All the changes in this PR are safe refactorings, and I didn't want to take on any risk. :smile: